### PR TITLE
Scrapyard 2 electric boogaloo

### PR DESCRIPTION
**Pull request by zombiemimic**
## About The Pull Request
Adds in the Scrapyard, a barren rock filled with derelict ships and angry aliens! Can you fight your way off, or will you become stranded and fated for xeno food?
image

Look for the hobo or prison guard to join the fun!

## Why It's Good For The Game
Allows more breadth in the Eris experience

## Changelog
:cl:
add: Added Scrapyard map
add: Added shuttle that goes from scrapyard to the Eris
add: Added more mapping things for the Scrapyard
/:cl:

### CODE REVIEW

**Comment by Smiruk** [comment]
So uhhh basically the ancient ghost roles station from tg 2.0?

File: code/game/objects/effects/spawners/ghostroles.dm [hunk 1]
```diff
@@ -68,3 +61,40 @@
 	r_pocket = /obj/item/ammo_magazine/sllrifle
 	id_slot = slot_wear_id
 	id_type = /obj/item/card/id
+
+/obj/effect/mob_spawn/human/hermit
+	name = "modified storage sleeper"
+	desc = "An old sleeper, with an sleeping body inside. The sleeper seems to have been gutted and padded into a bed."
```
Comment (Hopekz) [inline code/game/objects/effects/spawners/ghostroles.dm:67]:
```suggestion
	desc = "An old sleeper, with a sleeping body inside. The sleeper seems to have been gutted and padded into a bed."
```

File: maps/encounters/scrapyard/scrapyard.dm [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+	is_player_level = TRUE
+	is_contact_level = TRUE
+	is_accessable_level = TRUE
+	height = 1
+
+obj/effect/overmap/sector/scrapyard
+	name = "scrapyard"
+	desc = "An old spaceship graveyard."
+	generic_waypoints = list()
+	known = 1
```
Comment (hyperioo) [inline maps/encounters/scrapyard/scrapyard.dm:14]: Missing new line at the end

File: code/game/objects/effects/spawners/ghostroles.dm [hunk 3]
```diff
@@ -68,3 +61,40 @@
...
+		STAT_TGH = 24,
+		STAT_BIO = 8,
+		STAT_MEC = 12,
+		STAT_VIG = 16,
+		STAT_COG = 8
+	)
+
+/*obj/effect/mob_spawn/human/hermit/Initialize(mapload)
+	var/list/hermit_title = list("HTU Canterbury Longshoreman", "NSS Rocinante Marine", "MHS Geary Amputator 3rd Class", "HTU Epstein Bartender", "HTU Lehrer Miner")
+	title = "[pick(hermit_title)]"*/
```
Comment (hyperioo) [inline code/game/objects/effects/spawners/ghostroles.dm:88]: Do you need that? Usually we don't leave commented code to be merged.

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**